lib/generators/spotlight/install_generator.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable ClassLength
```

Comment:
Do you want to just disable `Metrics/ClassLength` for all generators? Or, maybe there's some sub-generators we ought to break out of here?